Engine/ScriptAnalyzerEngine.csproj

```diff
@@ -81,6 +89,7 @@
     <Compile Include="VariableAnalysisBase.cs" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="packages.config" />
```

Comment:
I think this file will be automatically added once the user downloaded the NuGet assembly.